requirements-windows.txt

```diff
@@ -0,0 +1,3 @@
+cx_Freeze==6.5.3
+cx-Logging==3.0
+psutil==5.8.0
```

Comment:
This dependency is not required, see further. It's only needed when child processes need to be killed.